cmd/gossamer/commands/account_test.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2023 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package commands
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// TestAccountGenerate test "gossamer account generate"
+func TestAccountGenerate(t *testing.T) {
+	testDir := t.TempDir()
+	directory := fmt.Sprintf("--keystore-path=%s", testDir)
+
+	AccountCmd.SetArgs([]string{"generate", directory})
+	err := AccountCmd.Execute()
+	require.NoError(t, err)
+}
+
+// TestAccountGeneratePassword test "gossamer account generate --password=VerySecurePassword"
+func TestAccountGeneratePassword(t *testing.T) {
+	testDir := t.TempDir()
+	directory := fmt.Sprintf("--keystore-path=%s", testDir)
+	AccountCmd.SetArgs([]string{"generate", directory, "--password=VerySecurePassword"})
+
+	err := AccountCmd.Execute()
+	require.NoError(t, err)
+}
+
+// TestAccountGenerateEd25519 test "gossamer account generate --scheme=ed25519"
+func TestAccountGenerateEd25519(t *testing.T) {
+	testDir := t.TempDir()
+	directory := fmt.Sprintf("--keystore-path=%s", testDir)
+	AccountCmd.SetArgs([]string{"generate", directory, "--scheme=ed25519"})
+
+	err := AccountCmd.Execute()
+	require.NoError(t, err)
+}
+
+// TestAccountGenerateSr25519 test "gossamer account generate --scheme=sr25519"
+func TestAccountGenerateSr25519(t *testing.T) {
+	testDir := t.TempDir()
+	directory := fmt.Sprintf("--keystore-path=%s", testDir)
+	AccountCmd.SetArgs([]string{"generate", directory, "--scheme=sr25519"})
+
+	err := AccountCmd.Execute()
+	require.NoError(t, err)
+}
+
+// TestAccountGenerateSecp256k1 test "gossamer account generate --scheme=secp256k1"
```

Comment:
IMHO I really like these comments, nice job making things clear!

Reply:
Thank you!